vpr/src/route/rr_graph_generation/interposer_cut.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <vector>
+#include "rr_graph_fwd.h"
+#include "rr_graph_view.h"
+#include "device_grid.h"
+
+/**
+ * @brief Goes through all edges in the RR Graph and returns a list of the edges that cross an interposer cut line.
+ * 
+ * @return std::vector<RREdgeId> List of all edges that cross an interposer cut line.
+ */
+std::vector<RREdgeId> mark_interposer_cut_edges_for_removal(const RRGraphView& rr_graph, const DeviceGrid& grid);
+
+/**
+ * @brief Shortens the channel nodes that cross an interposer cut line
```

Comment:
Explain what it does more (when a wire crosses an interposer cut line it is shortened to instead end at the interposer cut line, unless it is in the list of sg_node_indices that we should not cut).  Its drive point and the wire up to the interposer is retained.  Does this only work for unidirectional architectures / wires?  If so, say that.